projects/core/modules/notifications/notification-alert/notification-alert-options.ts

```diff
@@ -0,0 +1,22 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiNotification} from '@taiga-ui/core/enums';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface NotificationAlertOptions {
+    readonly autoClose: boolean | number;
+    readonly label: PolymorpheusContent<TuiContextWithImplicit<TuiNotification>>;
```

Comment:
It was designed to be plain string. Do you have a need to use templates there?

Reply:
Nope. I thought about ability to change label depending on status. But your idea is good. I've made it polymorpheus )

What do you think about adding an ability to change autoClose timeout depending on status also? How can we implement this with polymorpheus? Or, in this case, should we just use custom primitive and function type?

Reply:
@vladimirpotekhin @nsbarsukov @MarsiBarsi what do you think about adding `(status: TuiNotification) => number | boolean` type to `autoClose`? I can see it as both overengineering and a reasonable addition so people can, say, make errors stick while regular notifications would autoclose. What are your thoughts?